src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+
+import './Todo.scss';
+import { TodoType } from '../../types/TodoType';
+
+type Props = {
+  todo: TodoType;
+  onDelete: (todoId: number) => Promise<void>;
+  ids: number[];
+};
+
+export const Todo: React.FC<Props> = ({ todo, onDelete, ids }) => {
+  const [completed, setCompleted] = useState(todo.completed);
+  const [loading, setLoading] = useState(false);
```

Comment:
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/96409563/2a626569-f4db-4248-a8de-533e90b88eba)